laws/src/main/scala/cats/laws/ApplicativeLaws.scala

```diff
@@ -59,6 +59,9 @@ trait ApplicativeLaws[F[_]] extends ApplyLaws[F] {
   def applicativeUnit[A](a: A): IsEq[F[A]] =
     F.unit.map(_ => a) <-> F.pure(a)
 
+  def replicateAVoidReplicateA_Consistent[A](fa: F[A]): IsEq[F[Unit]] =
+    F.replicateA_(2, fa) <-> F.replicateA(2, fa).void
```

Comment:
I think we tried 5 and explosions resulted - if you think about this it makes sense as if the input data structure that is being replicated is already huge, replicateA_ will replicate it fivefold!

Reply:
How do you want to plumb this number though the laws? Should these laws accept an explicit parameter now? I don't think that would be binary compatible either.

Reply:
It's binary-compatible to add `n` as a parameter here, after all we are defining a brand new method! :)

What I'm suggesting is we leave the laws general, and do the hardcoding in the tests.

Reply:
@rabinarai1 I think if we can make `n` a parameter of the law then this should be good to go.

I think for the tests it would be good to use a value bigger than `n = 2`. To keep the structures from getting too large, I wonder if we can use scalacheck tricks like [`withSize`](https://github.com/typelevel/scalacheck/blob/78110e796a30fd77a0cc825363e191d89c32556c/core/shared/src/main/scala/org/scalacheck/Gen.scala#L485).

Reply:
@armanbilge we used Gen.resize to resize the generator of F[A]. But even increasing the number to 5 causes out of memory issues.